integration_tests/grpc_rest/model/helpers.go

```diff
@@ -193,6 +193,25 @@ func GetModel(
 	return &res, nil
 }
 
+func GetModelWithPathString(
```

Comment:
Better to name it `GetModelByHexVidPid`.

Reply:
Done! ✅